coriolis/osmorphing/redhat.py

```diff
@@ -295,7 +294,7 @@ def pre_packages_install(self, package_names):
         super(BaseRedHatMorphingTools, self).pre_packages_install(
             package_names)
         self._yum_clean_all()
-        if not self._has_package_installed('grubby'):
+        if not any('grubby' in package for package in self.installed_packages):
```

Comment:
As mentioned above, you can just check if 'grubby` is in `self.installed_packages`. Do please check the package name string as a whole, as this could have buggy results. Let's say your list has a package called 'grubby-somethhing', but you need 'grubby' package instead, this code would skip that install.